### PR TITLE
[Style] Specify file name

### PR DESCRIPTION
**Pull request by ignasi35**
It is very helpful to always indicate what file a snippet belongs to.

### CODE REVIEW

File: docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc [hunk 1]
```diff
@@ -25,6 +25,7 @@ The motivation for using gRPC as inter-service protocol is explained in the {akk
 
 First you should define the interface of the service in a protobuf service descriptor. It should be located in the `src/main/protobuf/ShoppingCartService.proto`.
 
+.src/main/protobuf/ShoppingCartService.proto
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc:28]:
```suggestion
.shopping-cart-service-scala/src/main/protobuf/ShoppingCartService.proto
```

File: docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc [hunk 2]
```diff
@@ -25,6 +25,7 @@ The motivation for using gRPC as inter-service protocol is explained in the {akk
 
 First you should define the interface of the service in a protobuf service descriptor. It should be located in the `src/main/protobuf/ShoppingCartService.proto`.
 
+.shopping-cart-service-scala/src/main/protobuf/ShoppingCartService.proto
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc:28]: In this case, I didn't remove the sentence immediately before the snippet but we could even remove it.

File: docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc [hunk 3]
```diff
@@ -46,6 +47,7 @@ sbt compile
 
 Let's implement the `ShoppingCartService` trait. Create a `ShoppingCartServiceImpl` class that extends the generated `ShoppingCartService` trait. Implement the `addItem` method from the trait by logging the call and return a successful `Future`.
 
+.shopping-cart-service-scala/src/main/scala/shopping/cart/ShoppingCartServiceImpl.scala
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc:50]: We take for granted users are familiar with the `src/main/scala/com/package` folder structure. Adding this metadata will prevent any confusion.

I just had to do a quick scan over the file copy/pasting the relevant bit from the `include::` macro below. Note that I stripped the `01-` prefix from the root folder.

**Comment by ignasi35** [comment]
I only did one page of the tutorial but I'd merge this PR already. If we crowd-source this change or apply the change as we edit other pages it should be enough.

**Review by octonato — COMMENTED** [review]
I don't know if it's a good idea. 

We are showing the path to file in the full project, but if you are following the guide using the seed template, the name of the project will be `shopping-cart-service`. 

> We take for granted users are familiar with the src/main/scala/com/package folder structure. Adding this metadata will prevent any confusion.

I think we can take that for granted and if not, people can learn new things. We can expect a minimal of autonomy from a developer.

File: docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc (hunk 1 above)
Comment (octonato) [reply docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc]: If we go for this option, I think we should not include `shopping-cart-service-scala`. Starting on `src` is good.

**Comment by ignasi35** [comment]
> We are showing the path to file in the full project, but if you are following the guide using the seed template, the name of the project will be `shopping-cart-service`.

Good catch. Fixed.

**Comment by ignasi35** [comment]
> I don't know if it's a good idea.

Fair enough. Let's see what others think. :-)

**Comment by patriknw** [comment]
I can see the value of this.
In some places we might want to adjust the text if we have this information anyway. For example "It should be located in the src/main/protobuf/ShoppingCartService.proto."

However, I wonder how this will look when we have Scala/Java tabs?

**Comment by ignasi35** [comment]
> However, I wonder how this will look when we have Scala/Java tabs?

Ah, good point. I don't know enough asciidoc or details about the solution Rich is putting together.

Let's put this on-hold.

**Comment by octonato** [comment]
We can add the pass as a comment inside the snippet at the first line. 

Also, it looks bit weird now. Looks like we forgot something in between. 

If we leave it outside, we can label it with something. Could be:

**Location:** `src/main/scala/shopping/cart/ShoppingCartServiceImpl.scala`

**Comment by ignasi35** [comment]
> If we leave it outside, we can label it with something. Could be:
> 
> **Location:** `src/main/scala/shopping/cart/ShoppingCartServiceImpl.scala`

Yeha, the good thing of using the syntax:

```
.path/filename.txt
[source, scala]
----
include...
----
```
Is that it's asciidoc syntax we can then theme however we like.

**Comment by ignasi35** [comment]
Screenshot 2020-10-08 at 11 27 25

**Comment by rstento** [comment]
> I don't know if it's a good idea.
> 
> We are showing the path to file in the full project, but if you are following the guide using the seed template, the name of the project will be `shopping-cart-service`.
> 
> > We take for granted users are familiar with the src/main/scala/com/package folder structure. Adding this metadata will prevent any confusion.
> 
> I think we can take that for granted and if not, people can learn new things. We can expect a minimal of autonomy from a developer.

True, @octonato, but in a tutorial we want to do everything we can to prevent them from making silly mistakes. I believe it is a good thing to show the filename with path. If they don't need to see it, they can ignore it.

**Comment by rstento** [comment]
> > If we leave it outside, we can label it with something. Could be:
> > **Location:** `src/main/scala/shopping/cart/ShoppingCartServiceImpl.scala`
> 
> Yeha, the good thing of using the syntax:
> 
> ```
> .path/filename.txt
> [source, scala]
> ----
> include...
> ----
> ```
> 
> Is that it's asciidoc syntax we can then theme however we like.

Yes, I think that the default styling of the title for a code block is a bit too big and too bold, and I don't like the italics. We can ask @rasummer to style it for us. I will try a few things and let you know.

**Comment by ignasi35** [comment]
Once discussed and agreed, I'm closing this (obsolete PR) in favor of Tracking it as a new issue in #363

**Pull request closed without merging**